arch/arm/src/armv7-r/arm_cache.c

```diff
@@ -81,14 +81,8 @@ void up_invalidate_dcache(uintptr_t start, uintptr_t end)
 
 void up_invalidate_dcache_all(void)
 {
-#ifdef CONFIG_ARCH_L2CACHE
```

Comment:
why `#ifdef CONFIG_ARCH_L2CACHE` is removed?

Reply:
Because if CONFIG_ARCH_L2CACHE isn't defined, all functions become empty macros:
https://github.com/apache/incubator-nuttx/blob/master/arch/arm/src/armv7-a/l2cc.h#L232-L240